cpp/perspective/src/cpp/table.cpp

```diff
@@ -122,16 +137,49 @@ Table::get_computed_schema(
                     type = computed_column_types[name_idx];
                 }
             } else {
-                type = schema.get_dtype(input_column);
+                type = master_table_schema.get_dtype(input_column);
             }
             input_types.push_back(type);
         }
 
+        if (skip) {
+            // this column depends on a column that does not exist, so it
+            // does not need to be typechecked.
+            continue;
```

Comment:
Why though?  With the removal protection also added in this PR - this case is an error, the column data may even be stale if for some reason it exists without its dependents?